tests/test_processing_common.py

```diff
@@ -86,9 +86,16 @@ def get_processor(self):
         processor = self.processor_class(**components, **self.prepare_processor_dict())
         return processor
 
+    def prepare_text_inputs(self, batch_size=None):
+        if batch_size is not None and batch_size > 1:
+            return ["lower newer", "upper older longer string"] + ["lower newer"] * (batch_size - 2)
+        return "lower newer"
+
     @require_vision
-    def prepare_image_inputs(self):
+    def prepare_image_inputs(self, batch_size=None):
         """This function prepares a list of PIL images for testing"""
+        if batch_size:
```

Comment:
Same here - it's better that I can specify for a batch size of any positive integer. Arguably even 0 would be OK as an input (that should raise an error)

```suggestion
        if batch_size is not None:
```